### DISCUSSION TITLE
PYTHON SERVER :: Server error -> Mailing section

### DISCUSSION THREAD

**Discussion by leen-kd** [saran-surya/email_auth_node#62] [Q&A]
Hello,
Currently I'm facing an issue with the email : 
flutter: email-auth >> Failed to send OTP ❌
flutter: email-auth >> Message from server :: (554, b'5.2.0 STOREDRV.Submission.Exception:OutboundSpamException; Failed to process message due to a permanent exception with message [BeginDiagnosticData]WASCL UserAction verdict is not None. Actual verdict is Suspend, ShowTierUpgrade. OutboundSpamException: WASCL UserAction verdict is not None. Actual verdict is Suspend, ShowTierUpgrade.[EndDiagnosticData] [Hostname=DB9P192MB1561.EURP192.PROD.OUTLOOK.COM]')
PYTHON SERVER :: Server error -> Mailing section

I already log in in my email and unblock it since they did but still can't send an otp code.

**Comment by saran-surya** [comment]
I guess you have used up your outlook max mailing limit, It says 
```Actual verdict is Suspend, ShowTierUpgrade.[EndDiagnosticData] [Hostname=DB9P192MB1561.EURP192.PROD.OUTLOOK.COM]')```

Please check with solutions regarding this issue and update in the issue, Should help other people also..

**Reply by saran-surya** [reply]
Hi Are you still facing the issue, If so kindly comment back on the discussion to re-open it.
Currently marking it as ***STALE***
